labels/de-DE.json

```diff
@@ -669,6 +669,7 @@
   "submitForApproval": "Zur Genehmigung einreichen",
   "submittedBy": "Eingereicht von",
   "subtotal": "Zwischensumme",
+  "subscriptionErroredBlob1": "Bonjour, <br/><br/> Der Status Ihres Abonnements Nr. {0} wird auf {1} aktualisiert. Das Abonnement wurde aufgrund eines möglichen Problems mit der Zahlung oder eines anderen Problems in einen fehlerhaften Zustand verschoben. Bitte wenden Sie sich für weitere Einzelheiten an den Kundendienst. Wenn Sie Ihr Abonnement anzeigen oder verwalten möchten, besuchen Sie bitte <a href=\"https://{2}/myaccount\">Mein Konto</a>.",
```

Comment:
Looks like you've used the French hello instead of German. Google translate gives me "Hallo" which I've used.

Reply:
german is correct, this is german locale file. see the one above "Zwischensumme" means "subtotal" in german.

Reply:
fixed it.